lib/pbench/cli/agent/commands/results/push.py

```diff
@@ -36,6 +36,13 @@ def execute(self) -> int:
     envvar="PBENCH_ACCESS_TOKEN",
     help="pbench server authentication token (will prompt if unspecified)",
 )
+@click.option(
```

Comment:
I think there is a way to avoid duplicating this option definition between move and push, but I am not sure exactly how to do it.

Reply:
removed the duplicated code.